deploy/helm/kubecf/assets/operations/instance_groups/routing-api.yaml

```diff
@@ -28,3 +31,35 @@
 {{- range $path, $bytes := .Files.Glob "assets/operations/pre_render_scripts/routing-api_*" }}
 {{ $root.Files.Get $path }}
 {{- end }}
+
+{{- else }}
+# Disable routing-api.
+- type: remove
+  path: /instance_groups/name=api/jobs/name=routing-api
+
+- type: remove
+  path: /instance_groups/name=database?/jobs/name=pxc-mysql/properties/seeded_databases/name=routing-api
+
+- type: replace
+  path: /instance_groups/name=router/jobs/name=gorouter/properties/routing_api/enabled
+  value: false
+
+- type: remove
+  path: /addons/name=bosh-dns-aliases/jobs/name=bosh-dns-aliases/properties/aliases/domain=routing-api.service.cf.internal
+
+- type: remove
+  path: /variables/name=routing_api_tls
+
+- type: remove
+  path: /variables/name=routing_api_tls_client
+
+# Disable the tcp-router ig, as there will be no routing-api SSE events to read from.
+- type: remove
+  path: /instance_groups/name=tcp-router
```

Comment:
Shouldn't this live in `tcp-router.yaml`?

Reply:
yes indeed, @mook-as pls request changes, Im rerequesting you a review.!

Reply:
Request changes it is then!